app/views/art_items/_form.html.erb

```diff
@@ -66,7 +66,7 @@
   </div>
 
   <div id="attached_files" class="my-5">
-    <%= form.label :documents, "Documents (pdf,txt,jpg,png,doc)" %><span class="optional-text"> * optional</span>
+    <%= form.label :documents %><span class="optional-text">(5 Mb limit per item) * optional</span>
```

Comment:
I think lower case _b_ in "Mb" is for bit. Upper case _B_ is for Byte.

Reply:
B dun